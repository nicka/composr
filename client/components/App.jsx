import React from 'react';
import {
  BrowserRouter as Router,
  Route,
  Switch,
} from 'react-router-dom';
import Home from './Home';
import Faq from './Faq';
import PageNotFound from './PageNotFound';
import ComposrPlusBandlab from './../assets/composr-bandlab.png';

const App = () => (
  <Router>
    <div>
      <img alt="BandLab Welcomes Composr" height="80" src={ComposrPlusBandlab} />
      <Switch>
        <Route exact path="/" component={Home} />
        <Route path="/faq" component={Faq} />
        <Route render={PageNotFound} />
      </Switch>
      Â© 2017 Composr
    </div>
  </Router>
);

export default App;
