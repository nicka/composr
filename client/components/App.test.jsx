import React from 'react';
import renderer from 'react-test-renderer';
import App from './App';

describe('App', () => {
  it('renders the component', () => {
    const component = renderer.create(<App />);
    expect(component.toJSON()).toMatchSnapshot();
  });
});
