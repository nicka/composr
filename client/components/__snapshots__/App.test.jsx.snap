// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders the component 1`] = `
<div>
  <img
    alt="BandLab Welcomes Composr"
    height="80"
    src="test-file-stub"
  />
  <div>
    <h1>
      Page not found
    </h1>
    <p>
      <a
        href="/"
        onClick={[Function]}
      >
        Back to announcement
      </a>
    </p>
  </div>
  Â© 2017 Composr
</div>
`;
